readme.md

```diff
@@ -8,43 +7,42 @@ The general awkwardness of DOM functions is even worse for SVG than for HTML.
 
 You don't want to create an element with:
 
-    var rect = document.createElementNS("http://www.w3.org/2000/svg", "rect");
-	
+    const rect = document.createElementNS("http://www.w3.org/2000/svg", "rect");
+
 You'd rather have concise, chainable and convenient functions not too inspired by Java, and to have internal references to gradients managed for you. Of course you want the few traps and browser differences handled for you.
 
 And that might be about all you need. Because SVG is already a very high level model, you don't usually need a higher model to draw shapes in your browser. You don't need functions hiding the logic of the SVG.
 
-And you don't want to import a big JavaScript file for that. [hu.min.js](hu.min.js) weights 1.3 kb when gziped and has no dependency.
+And you don't want to import a big JavaScript file for that. [hu.min.js](hu.min.js) weights 1.4 kb when gziped and has no dependency.
 
 That's why you might find hu.js useful, assuming its limited set of features isn't too limited for you.
 
 # Basic Example
 
-
     // Add to the body a new SVG element
-    var svg = hu('<svg>', 'body').css({ width:100, height:100 });
-    
+    const svg = hu('<svg>', 'body').css({ width:100, height:100 });
+
     // Define a reusable radial gradient
-    var grad = svg.rgrad(0.3, 0.3, 0.8, 'yellow', 'green');
-    
+    const grad = svg.rgrad(0.3, 0.3, 0.8, 'yellow', 'green');
+
     // Draw a circle using this gradient
     hu('<circle>', svg).attr({ cx:50, cy:50, r:50, fill:grad });
 
 ### Result :
 
-![](http://dystroy.org/demos/hu.js/hu-sphere.png)
+<img src="./demo.svg" width="100" height="100" alt="Result of the demo, a green ball with yellow sheen" />
```

Comment:
Why this precise change ?

Reply:
Before, it showed a rendered PNG of the SVG, but now because GitHub supports rendering SVGs directly in a markdown file (as long as you link to it), I figured it would be better to show the actual SVG :)

Reply:
Oh, right